lib/tasks/db.js

```diff
@@ -165,6 +165,35 @@ module.exports = {
     }
   },
 
+  'db:migrate:status': {
+    descriptions: {
+      'short': 'List the status of all migrations'
+    },
+
+    task: function () {
+      return getMigrator('migration').then(function (migrator) {
+        return ensureCurrentMetaSchema(migrator).then(function () {
+          return migrator.executed();
+        }).then(function (migrations) {
+          _.forEach(migrations, function (migration) {
```

Comment:
I don't like the way you are logging the results. Maybe something like this would be nicer:
```
Executed migrations:
- <migration file 1>
- <migration file 2>
...

Pending migrations:
- <migration file 3>
- <migration file 4>
...
```

If there are no executed or pending migrations, instead of that just log:

```
No executed migrations found.

No pending migrations found.
```